src/core/include/openvino/op/util/attr_types.hpp

```diff
@@ -155,8 +155,8 @@ std::ostream& operator<<(std::ostream& s, const TopKMode& type);
 
 /// \brief Implicit broadcast specification
 struct OPENVINO_API AutoBroadcastSpec {
-    AutoBroadcastSpec() : m_type(AutoBroadcastType::NONE), m_axis(0) {}
-    AutoBroadcastSpec(AutoBroadcastType type) : m_type(type), m_axis(0) {}
+    AutoBroadcastSpec() : m_type(AutoBroadcastType::NONE), m_axis(-1) {}
+    AutoBroadcastSpec(AutoBroadcastType type) : m_type(type), m_axis(-1) {}
```

Comment:
Is there any chance to break backward compatibility because of the default value change?

Reply:
yes, we indeed found some previous CI tests(some are in other components and plugins) failed if change this default value. But I still think using `-1` as default value making more sense for `BroadcastType::PDPD` based on its specs. what about using `-1` for PDPD, and keep `0` for another `BroadcastType`?
For the CI failed issues, I will try to fix them in this PR(maybe in next commit, so currently will convert this PR to draft) if no need to modify kernel implement of other components and plugins.

Reply:
Actually the `axis` attribute is used only for `BroadcastType::PDPD`, so it shouldn't affect other broadcast types.
Anyway current solution lgtm.